### PR TITLE
Add built target for fedora

### PR DESCRIPTION
**Pull request by KjellMorgenstern**

### CODE REVIEW

File: docker/Dockerfile.fedora [hunk 1]
```diff
@@ -0,0 +1,23 @@
+FROM fedora:31
+
+ARG user_id=1000
+ENV env_user_id=${user_id}
+
+RUN groupadd -g ${env_user_id} conan && \
+    useradd -r -u ${env_user_id} -g conan conan
+
+USER root
```
Comment (ovidiub13) [inline docker/Dockerfile.fedora]: You are root at this point. There is no need for this change.

File: docker/Dockerfile.fedora [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+WORKDIR /home/conan
```
Comment (ovidiub13) [inline docker/Dockerfile.fedora]: This directory needs to exist in order to change to it.

File: docker/Dockerfile.fedora [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+RUN dnf -y install openssl-devel zlib-devel libjpeg-devel libpng-devel bzip2 libgit2-devel boost-devel
+RUN dnf -y install xorg-x11-server-Xvfb qt-creator qt5-devel
+
+RUN ln -s /usr/bin/qmake-qt5 /usr/bin/qmake
+RUN ln -s /usr/bin/lrelease-qt5 /usr/bin/lrelease
+
+RUN dnf clean all
+
+USER conan
```
Comment (ovidiub13) [inline docker/Dockerfile.fedora]: Please add trailing end line.

File: docker/Dockerfile.fedora (hunk 2 above)
Comment (KjellMorgenstern) [reply docker/Dockerfile.fedora]: I added the -m option to the useradd command

**Pull request merged**